core/server/master/src/main/java/alluxio/master/file/meta/TtlBucketList.java

```diff
@@ -46,7 +46,7 @@ public TtlBucketList() {
    * @return the bucket containing the inode, or null if no such bucket exists
    */
   private TtlBucket getBucketContaining(Inode<?> inode) {
-    if (inode.getTtl() == Constants.NO_TTL) {
+    if (inode.getTtl() <= Constants.NO_TTL) {
```

Comment:
Prefer the original or `inode.getTtl() < 0`.

Reply:
what's the different between **inode.getTtl() < 0** and **inode.getTtl() <= -1 (Constants.NO_TTL)**

Reply:
They are the same today, but using `inode.getTtl() <= Constants.NO_TTL` could lead to a bug later. It adds an additional constraint on `Constants.NO_TTL` that all values representing "no ttl" must be less than or equal to `Constants.NO_TTL`. Previously we could change `Constants.NO_TTL` to `-10` or `MIN_INT` without any consequences.

On further thought, I think it's better to stick with `inode.getTtl() == Constants.NO_TTL` and use `-1` only to mean "no ttl". I don't think it's very helpful to users to be able to use `-5`, and it's just simpler for us to have a single `Constants.NO_TTL` value which means "no ttl".